frontend/src/views/Landing/index.tsx

```diff
@@ -25,6 +27,76 @@ import SingleCellDataIconActive from "./icons/single-cell-data-active";
 import SingleCellDataIconInactive from "./icons/single-cell-data-inactive";
 import styles from "./index.module.scss";
 
+const DEFAULT_NUM_CELLS = 33000000;
+const DEFAULT_NUM_DATASETS = 436;
+const DEFAULT_NUM_CELL_TYPES = 520;
+
+const COUNT_UP_DURATION = 1.5;
+
+const HeroStatsCounter = ({
+  className,
+}: {
+  className: string;
+}): JSX.Element => {
+  const [hasUpdated, setHasUpdated] = useState(false);
+  const corpusSummary = useCorpusSummary();
+  const formatNumber = useCallback((n: number): string => {
```

Comment:
since this function doesn't depend on any component specific variable, we could just extract it as a function outside of the component?

E.g.,

```tsx
const HeroStatsCounter: () => {
   ...
  // use formatNumber
}

function formatNumber(n: number) {
   ...
}
```